next-app/src/routes/projects/[project_code]/meta/+server.ts

```diff
@@ -44,7 +44,7 @@ function has_audio(entry) {
 	//			'...-audio': '...'
 	//		}
 	// }
-	const in_fields = fields => Object.keys(fields).some(name => Object.keys(fields[name]).some(contains_audio))
+	const in_fields = (fields: Field[]) => Object.keys(fields).some(name => Object.keys(fields[name]).some(contains_audio))
```

Comment:
`Field[]` is not the correct type here. The value that you are passing in when calling this method is an object and you're using at as an object when you call the keys function. As such I would use `Record<string, Field>`.

Reply:
let's discuss this if we can get together

Reply:
sounds good, I sent you a slack.